doc/user-guide/MongooseIM-High-level-Architecture.md

```diff
@@ -0,0 +1,59 @@
+## Inside MongooseIM
+
+MongooseIM comes with a large set of modules to configure and enable. See 'Extension Modules' for more info.
+
+MongooseIM manages two sets of data: transient for session data management, and persistent for archive and configurations.
+
+## MongooseIM and its ecosystem in a datacenter
+
+### Databases
```

Comment:
I'm not able to understand what the author had in mind in this section :) Maybe the formatting is wrong or I miss ed sth.